metadata.yml

```diff
@@ -1,7 +1,7 @@
 rules:
   - apiGroups: [""]
     apiVersions: ["v1"]
-    resources: ["pods"]
+    resources: ["deployment", "statefulset"]
```

Comment:
I would keep pods, and probably add all the other supported resources 
Deployment, ReplicaSet, StatefulSet, DaemonSet, ReplicationController, Job, CronJob, Pod

Reply:
Done.